install_ffmpeg.sh

```diff
@@ -16,15 +16,13 @@ fi
 if [ ! -e "$HOME/x264/x264" ]; then
   git clone http://git.videolan.org/git/x264.git "$HOME/x264"
   cd "$HOME/x264"
-  # git master as of this writing
-  git checkout 7d0ff22e8c96de126be9d3de4952edd6d1b75a8c
```

Comment:
For now, let's keep checking out a consistent hash because of the bit-exact verification requirement. Once we're confident in non-determinisic verification then we work directly from master.

Reply:
fixed